config/config.go

```diff
@@ -332,7 +332,7 @@ func (c *GlobalConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
-// isZero returns true iff the global config is the zero value.
+// isZero returns true if the global config is the zero value.
```

Comment:
`iff` = `if and only if`. Same for the other `iff` in this PR. So not need to change them.

Reply:
ok.I will close this PR.